.github/workflows/publish.yml

```diff
@@ -11,34 +11,78 @@ on:
     paths:
       - "src/**"
       - ".github/workflows/publish.yml"
+  repository_dispatch:
+    types: [create-release]
+    client_payload:
+      release_tag:
+        required: true
 
 jobs:
+  create_release:
+    if: ${{ github.event_name == 'repository_dispatch' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: Brightspace/third-party-actions@actions/checkout
```

Comment:
do we even need to check out the repo in this job?

Reply:
Hmm I don't think so. It might have been needed when I had it as a separate action in a different file